src/core/util/target_loader.py

```diff
@@ -55,23 +55,25 @@ def collect_tests():
                         exclude_params = exclude.split(',')
                     current_full_path = os.path.join(dir_path, current_file)
                     if current_file.endswith('.py') and not current_file.startswith('__'):
-                        if include is '' and exclude is '' and directory is '':
-                            if not current_full_path in test_list:
+                        if include == '' and exclude == '' and directory == '':
+                            if not current_full_path not in test_list:
```

Comment:
This seems like a very odd conversion. The original is a negative statement, but the suggested replacement is a double negative which seems like it would give the wrong answer.

Reply:
oops, that first not should have been deleted